chef/cookbooks/crowbar/templates/default/crowbar-reload-nova-after-upgrade.sh.erb

```diff
@@ -27,21 +27,41 @@ if [[ -f $UPGRADEDIR/crowbar-reload-nova-after-upgrade-ok ]] ; then
     exit 0
 fi
 
+# Check the python-nova package to make sure it is at the latest version with
+# no more recent versions in any zypper repositories (this package can be used
+# as a proxy for making sure all Nova service packages are up to date since
+# they are version locked to python-nova):
```

Comment:
How could this happen? Aren't we responsible for updating the package? Repositories should be set up and the package was updated in different step, right?

Reply:
We are, but the `zypper --non-interactive up openstack-nova-compute` we run for the early `nova-compute` upgrade we require for RPC API compatibility is not trustworthy. For instance, it exited `0` when we'd forgotten to set `solver.allowVendorChange` earlier, thus causing the installation to fail with a prompt asking about vendor change. I don't know how `zypper` behaves with other problems, such as a NIC flap preventing access to the package repository, but I'd rather play it safe and check up on it.

Reply:
OK, then. It does not hurt to have another check I guess